packages/core/src/RenderingEngine/RenderingEngine.ts

```diff
@@ -959,28 +1022,18 @@ class RenderingEngine {
   } {
     const { offScreenCanvasContainer, offscreenMultiRenderWindow } = this;
 
-    // 1. Calculated the height of the offScreen canvas to be the maximum height
-    // between canvases
-    const offScreenCanvasHeight = Math.max(
-      ...canvasesDrivenByVtkJs.map((canvas) => canvas.height)
-    );
-
-    // 2. Calculating the width of the offScreen canvas to be the sum of all
-    let offScreenCanvasWidth = 0;
-
-    canvasesDrivenByVtkJs.forEach((canvas) => {
-      offScreenCanvasWidth += canvas.width;
-    });
-
     // @ts-expect-error
-    offScreenCanvasContainer.width = offScreenCanvasWidth;
+    offScreenCanvasContainer.width = MAX_WIDTH;
```

Comment:
If you set this after checking the packing you can significantly reduce memory usage - otherwise I suspect you will crash many browsers.